test/tests/jade.js

```diff
@@ -6,35 +6,21 @@ var path = require('path');
 var jsxgettext = require('../../lib/jsxgettext');
 var jade = require('../../lib/parsers/jade').jade;
 
+var utils = require('../utils');
 
 exports['test parsing'] = function (assert, cb) {
-  function assertMsg(result, msg) {
-    msg = '"' + msg + '"';
-    assert.ok(result.indexOf('msgid ' + msg) > -1, msg + ' is found');
-  }
-
-  // check that files with leading hash parse
-  var inputFilename = path.join(__dirname, '..', 'inputs', 'second_attribute.jade');
+  var inputFilename = path.join(__dirname, '..', 'inputs', 'example.jade');
   fs.readFile(inputFilename, "utf8", function (err, source) {
-
     var opts = {keyword: ['gettext', '_']},
-        sources = {'inputs/second_attribute.jade': source},
```

Comment:
I was actually thinking about it. But there is a second test that is using `second_attribute.jade` - and I was not entirely comfortable just removing it.
Any idea what exactly is tested here - seems to be related to #40
ping @mweibel